ichnaea/content/templates/api.pt

```diff
@@ -60,6 +60,12 @@
         service.
```

Comment:
Should the paragraph above also be commented out?

Reply:
So I thought about that, I'm not sure it should be removed only b/c it describes how the api functions wrt the app itself.